Clean up links

oddhack opened the pull request:
* Rewrite to current registry locations, instead of getting redirected from old locations through .htaccess
* Remove leading part of khronos.org URL in the places I was otherwise modifying links, for consistency (didn't do this in all cases)

@Khronoswebmaster there are a lot of link changes here, and I can't verify them since
they're relative references inside PHP files. So I strongly advise
running a link-checker over the blue boxes and resource pages
immediately after accepting the MR. But the changes are straightforward,
unless I made some really obvious consistent blunder.

Khronoswebmaster commented:
It might be more sensible to change the links to include https://www.khronos.org/ instead of removing the domain, so that links will still work on Github or if someone clones Khronosdotorg locally. Granted, I had not fully converted to include the domain when you made these changes.

oddhack commented:
Pity there isn't a base URL setting in .md (or maybe there is?).

oddhack commented:
Try now. I added the prefix to a bunch of relative URLs I hadn't otherwise touched in the previous commit, too, and changed some http -> https (there are a zillion more http URLs in there - I run HTTPS Everywhere so don't really care, but others might).
